package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "lint": "eslint \"src/**/*.js\" \"test/**/*.js\" \"examples/**/*.js\"",
     "doc": "jsdoc src/Core/Commander/Interfaces/ApiInterface/*",
-    "doclint": "npm run doc -- -t templates/silent",
+    "doclint": "npm run doc -- -t ../minami",
```

Comment:
So I guess, you would like me to do something like this : 

```
"doc": "jsdoc src/Core/Commander/Interfaces/ApiInterface/* -t node_modules/minami/",
"doclint": "npm run doc -- -t templates/silent",
```

I guess the part  `-t node_modules/minami/` could be put in `autodeploy.sh` too, I don't know what is the best.